modules/setting/log.go

```diff
@@ -131,7 +131,7 @@ func generateLogConfig(sec *ini.Section, name string, defaults defaultLogOptions
 		logConfig["username"] = sec.Key("USER").MustString("example@example.com")
 		logConfig["password"] = sec.Key("PASSWD").MustString("******")
 		logConfig["host"] = sec.Key("HOST").MustString("127.0.0.1:25")
-		logConfig["sendTos"] = sec.Key("RECEIVERS").MustString("[]")
+		logConfig["sendTos"] = strings.Split(sec.Key("RECEIVERS").MustString(""), ",")
```

Comment:
~~What about a regexp? That could suport several kinds of syntaxes. What is it that we expect here? The standard full-name format for an e-mail is:
`"John Snow" <john@thewall.com>`. Entries are usually separated by `;` but `,` could also be supported.~~

Anyway, we should validate the addresses and avoid using anything that _will make the server fail_.

Validation, however is something [really tricky](https://medium.com/hackernoon/the-100-correct-way-to-validate-email-addresses-7c4818f24643).

Reply:
Validating email addresses is extremely non-trivial as just about the only thing you have to have is @

Even splitting potential lists of them by "," is incorrect as "," can be in there too.

In any case it can't blow up the server - SendMail will simply return an error and any error whilst logging an event just ends being printed blandly to stdout. 

See modules/log/event.go

Reply:
Mmm.... not even `TrimSpace()` on each value? Anyway, this is a closed list, so admins should be responsible and use whatever Gitea supports (i.e. not e-mails with `,` in it 😄).

Reply:
OK, I've added the TrimSpace. If you start looking too hard we'll have to fix the:

https://github.com/go-gitea/gitea/blob/802aa6d5f306ade08438dc07d93913daecb1d3ab/modules/log/smtp.go#L84

And make it do proper quoting of email addresses - but I think that could be done in a different PR.